packages/derive/src/callable_points.rs

```diff
@@ -0,0 +1,100 @@
+use proc_macro2::Span;
+use proc_macro2::TokenTree;
+use quote::quote;
+use serde::ser::{Serialize, SerializeMap, Serializer};
+
+use crate::callable_point;
+
+pub fn make_callable_points(body: Vec<syn::Item>) -> (Vec<TokenTree>, Vec<(String, bool)>) {
+    let mut list_callable_points = Vec::new();
+    let mut made = Vec::new();
+
+    for i in &body {
+        if let syn::Item::Fn(function) = i.clone() {
+            let (function_remove_macro, is_stripped) = strip_callable_point(function);
+
+            if is_stripped {
+                let (made_callable_point, list_callable_point) =
+                    callable_point::make_callable_point(function_remove_macro);
+                made.extend(made_callable_point);
+                list_callable_points.push(list_callable_point);
+            } else {
+                made.extend(quote! {#i});
+            }
+        } else {
+            made.extend(quote! {#i});
+        }
+    }
+
+    (made, list_callable_points)
+}
+
+/// If the function has `#[callable_point]`, strip it.
+///
+/// example:
+///
+/// ```no_test
+/// #[callable_point]
+/// fn foo(deps: Deps, _env: Env, ...) -> u32 {
+///     42
+/// }
+/// ```
+///
+/// Output:
+///
+/// ```no_test
+/// fn foo(deps: Deps, _env: Env, ...) -> u32 {
+///     42
+/// }
+/// ```
+pub fn strip_callable_point(function: syn::ItemFn) -> (syn::ItemFn, bool) {
```

Comment:
Is the file used as follows?
- `callable_point.rs` is used when handling multiple functions.
- `callable_points.rs` is used when handling one function.

If so, shouldn't this function be defined in `callable_point.rs`?